spec/cypress/integration/admin/SchoolManagment.feature

```diff
@@ -9,6 +9,7 @@ Feature: Admin user managing schools
   Scenario: Viewing a list of schools
     Then the table should have 20 rows
     And the page should be accessible
+    And percy should be sent snapshot
```

Comment:
What happens when you don't name them? What does the name do?

Reply:
Defaults to the scenario+feature name. Basically we only need to specify the name if there's more than one snapshot per feature.